parm/atm/obs/config/satwind.yaml

```diff
@@ -13,226 +13,872 @@ obs space:
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 99.
+      maxvalue: 200.
+    test variables:
+    - name: MetaData/windComputationMethod
+    maxvalue: 4.
+    action:
+      name: reject
+  # Reject obs with qualityInformationWithoutForecast < 85.
+  # CLEARED
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 99.
+      maxvalue: 200.
+    test variables:
+    - name: MetaData/qualityInformationWithoutForecast
+    minvalue: 85.
+    action:
+      name: reject
+# NESDIS: satelliteIdentifier [250–299] (>249, <300)
+  # Reject obs with satelliteZenithAngle > 68 deg
+  # NOT EXPLICITLY CLEARED: No obs in this range in file, so 0 Bounds Check rejects (which is correct) but essentially untested
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 249.
+      maxvalue: 300.
+    test variables:
+    - name: MetaData/satelliteZenithAngle
+    maxvalue: 68.
+    action:
+      name: reject
+  # Reject obs with qualityInformationWithoutForecast < 90. OR > 100.
+  # CLEARED
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 249.
+      maxvalue: 300.
+    test variables:
+    - name: MetaData/qualityInformationWithoutForecast
+    minvalue: 90.
+    maxvalue: 100.
+    action:
+      name: reject
+  # Reject obs with air_pressure < 15000.
+  # CLEARED
   - filter: Bounds Check
     filter variables:
     - name: eastward_wind
     - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 249.
+      maxvalue: 300.
     test variables:
     - name: MetaData/air_pressure
-    minvalue: 70000
+    minvalue: 15000.
+    action: 
+      name: reject
+  # Reject obs with air_pressure < 70000. when Type=251
+  # NOT EXPLICITLY CLEARED: No obs in this range in file, so 0 Bounds Check rejects (which is correct) but essentially untested
+  - filter: Bounds Check
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
     where:
-    - variable:
-        name: ObsType/eastward_wind
-      is_in: 242, 243
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 249.
+      maxvalue: 300.
+    - variable: ObsType/eastward_wind
+      is_in: 251
+    test variables:
+    - name: MetaData/air_pressure
+    minvalue: 70000.
     action:
       name: reject
-  # MODIS-Aqua/Terra (257) and (259), reject when pressure less than 250 mb.
+  # Reject obs with air_pressure > 30000. when Type=246
+  # CLEARED
   - filter: Bounds Check
     filter variables:
     - name: eastward_wind
     - name: northward_wind
+    where:
+    - variable: MetaData/satelliteIdentifier
+      minvalue: 249.
```

Comment:
I've seen this a lot. Should satelliteIdentifier min and max values be floats?

Reply:
@nicholasesposito the `MetaData/satelliteIdentifier` variable is stored as a float, so I think the convention choice we were settling on is to assert these bounds as explicit float values. I think ideally these should be stored as integers in the diag file and we could change these Bounds Checks to using the `is_in` function instead of relying on `minvalue` and `maxvalue`.

Reply:
It's interesting that it's stored as a float, but if that's the case, then it is what it is. I yes, agreed that storing as integers would be preferred.